### DISCUSSION TITLE
Can receive h264 stream by libdatachannel?

### DISCUSSION THREAD

**Discussion by feiwei9696** [paullouisageneau/libdatachannel#966] [General]
I don't find the implementation for h264 depacketization, or just receive rtp packet without depacketization for h264?
@paullouisageneau 

Thanks.

**Comment by paullouisageneau** [comment]
There is not such module for now as most people use libdatachannel to send media (see the related issue https://github.com/paullouisageneau/libdatachannel/issues/676), you are welcome to contribute one! By default the track will receive RTP packets.
